### PR TITLE
Add information about building

### PR DESCRIPTION
**Pull request by sypets**
- Add information about location of SCSS and TypeScript files
- Add information about building
- Add information about linting

Resolves: #159

### CODE REVIEW

File: Documentation/Build/Index.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.. index::
+   single: Code Contribution Workflow; Build
+   single: Build
+   single: SCSS
+   single: TypeScript
+.. _build:
+
+=====
+Build
```
Comment (mbrodala) [inline Documentation/Build/Index.rst:13]: If we focus on assets then maybe **Building assets** would be a more descriptive title.

File: Documentation/Build/Index.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+=====
+
+This page explains building JavaScript and CSS files.
```
Comment (mbrodala) [inline Documentation/Build/Index.rst:16]:
```suggestion
This page explains building assets like JavaScript and CSS files.
```

File: Documentation/Build/Index.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+For the most part, the TypeScript, JavaScript and SCSS sources are contained in
+:file:`Build/Sources`. These are converted into target files, usually located in:
```
Comment (mbrodala) [inline Documentation/Build/Index.rst:19]:
```suggestion
:file:`Build/Sources`. These are compiled to target files, usually located in:
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/Build/Index.rst (hunk 1 above)
Comment (sypets) [reply Documentation/Build/Index.rst]: I pushed another commit to fix make suggested changes.

**Pull request merged**